straxen/analyses/pulse_plots.py

```diff
@@ -62,8 +62,16 @@ def plot_pulses(context, raw_records, run_id, time_range,
     records = strax.raw_to_records(raw_records)
     records = strax.sort_by_time(records)
     strax.zero_out_of_bounds(records)
+
```

Comment:
This is a bug fix which was needed due to the changes in https://github.com/XENONnT/straxen/pull/404/files . It only affected the nveto pulse plotting.